### PR TITLE
Removing cert self-sign option from operator

### PR DESCRIPTION
**Pull request by macxamin**
Webhooks with self-signed certs will not be accepted by the kubernetes API server.

### CODE REVIEW

**Comment by fabxc** [comment]
Can you run `make presubmit` and submit any potential changes as well?

**Comment by macxamin** [comment]
@fabxc ran `make presubmit`. No new files were generated.

File: cmd/operator/main.go [hunk 1]
```diff
@@ -73,14 +73,16 @@ func main() {
...
 		caSelfSign = flag.Bool("ca-selfsign", true,
 			"Whether to self-sign or have kube-apiserver sign certificate key pair for TLS.")
 		webhookAddr = flag.String("webhook-addr", ":8443",
 			"Address to listen to for incoming kube admission webhook connections.")
 		metricsAddr = flag.String("metrics-addr", ":18080", "Address to emit metrics on.")
 	)
 	flag.Parse()
-
+	// TODO(maxamin) Solves the "declared but not used", remove after GKE is updated.
+	_ = caSelfSign
```
Comment (fabxc) [inline cmd/operator/main.go:85]: Why is this necessary? We'd update the binary and the flags it is deployed with in one change,no? @pintohutch

Comment (pintohutch) [reply cmd/operator/main.go:85]: Yep - the GKE component is safe as we'll remove the option there after we tag and do a release here.

Comment (pintohutch) [reply cmd/operator/main.go:85]: So it's safe to delete this flag entirely here

**Review by fabxc — APPROVED** [review]
